csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -200,14 +200,17 @@
     </Exec>
 
     <PropertyGroup>
+      <CurrentDate>$([System.DateTime]::UtcNow.ToString(yyyyMMdd))</CurrentDate>
+      <CurrentTime>$([System.DateTime]::UtcNow.ToString(hhmm))</CurrentTime>
       <PackageVersion>@(MajorVersionNumber)</PackageVersion>
       <Version>$(PackageVersion)</Version>
-      <PackageVersion Condition="'$(IsReleaseBuild)'!='true'">$(PackageVersion)-dev-$(GitCommitHashShort)</PackageVersion>
+      <PackageVersion Condition="'$(IsReleaseBuild)'!='true'">$(PackageVersion)-dev-$(CurrentDate)-$(CurrentTime)-$(GitCommitHashShort)</PackageVersion>
```

Comment:
Does Visual Studio still recognize this format as a "dev" package? I'm assuming the sort order in VS will be improved where are are multiple dev packages available in the source location.

Reply:
yes, this is still recognized as dev package. VS or NuGet repo orders the text after the main n.n.n version number lexicographically, so, having an equal length string with datetime, will bring the latest on the top.